app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_result, description",
+    [
+        (14, 14, [0, 0], "Age less than 15 should return [0, 0]"),
+        (23, 23, [1, 1], "Age equals to 23 return [1, 1]"),
+        (24, 24, [2, 2], "When age 24 should return [2, 2]"),
+        (-1, -1, [0, 0], "Negative ages should return [0, 0]"),
+        (
+            17,
+            17,
+            [1, 1],
+            "When age equals to 15 and less than 24 should return [1, 1]",
```

Comment:
you should use `id/ids` in parametrizer for text description - it is not a separate parameter and shouldnt be passed to test function explicitly